site/content/contributing.md

```diff
@@ -1,3 +1,204 @@
+The AWS Deploy Tool for .NET is an open source project hosted on [GitHub](https://github.com/aws/aws-dotnet-deploy) and we openly welcome community contributions.
+
+[Click here](https://github.com/aws/aws-dotnet-deploy/compare) to submit a pull request.
+
+
+## Using the Deployment Tool from Source
+
+* Ensure that all the [pre-requisties](https://aws.github.io/aws-dotnet-deploy/docs/getting-started/pre-requisites/) (including Docker) are installed and your default AWS profile has admin access
+* Clone the repository using `git clone https://github.com/aws/aws-dotnet-deploy.git`
+* Open `AWS.Deploy.sln` in Visual Studio.
+* Ensure that `../src/AWS.Deloy.CLI` is set as the start up project.
```

Comment:
Should this `../src/AWS.Deloy.CLI` just be `src/AWS.Deloy.CLI`? I'm not sure why we are saying go up to the parent directory.

Reply:
Fixed